Graphics/PostEffects/Tonemap.py

```diff
@@ -0,0 +1,26 @@
+
+import arcade
+from ..PostProcessingChain import PostProcessingStage
```

Comment:
The "pythonic" way to handle imports is usually via absolute imports like:
```py
import game.Graphics.PostEffects.PostProcessingChain import PostProcessingStage
```

I have my IDE configured to do this automatically (PyCharm/IntelliJ) for me. This does help with making moving files easier too. The lines don't change if a file moves.

Reply:
How does the top level package get named 'game' ?

Reply:
In the case of the current app I think you'd just import `Graphics.PostEffects.PostProcessingChain`. If it included `game` then the path would be `<repo>/game/Graphics.PostEffects.PostProcessingChain`.

The base folder that Python imports from is the folder where you run the `python` executable.